sdk/src/types.rs

```diff
@@ -165,6 +168,127 @@ impl LocalAccount {
     }
 }
 
+/// Types of hardware wallet the SDK currently supports
+#[derive(Debug)]
+pub enum HardwareWalletType {
+    Ledger,
+}
+
+pub trait TransactionSigner {
```

Comment:
Follow up items - Update/refactor LocalAccount to impl this trait as well

issue #8719